cmd/neofs-cli/modules/storagegroup/root.go

```diff
@@ -25,6 +25,8 @@ const (
 	cidFlag   = "cid"
 )
 
+const sgLifetimeFlag = "lifetime"
```

Comment:
We already have `expires-on`, why not reuse it (together with `+N` format)?

Reply:
lets keep it as it is and change that for all the CLI commands in a separate commit: https://github.com/nspcc-dev/neofs-node/issues/1574